src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy

```diff
@@ -288,4 +289,156 @@ round trip latency rule is removed on the dst switch"() {
             }
         }
     }
+
+    @Tidy
+    @Tags([SMOKE_SWITCHES])
+    def "A round trip latency ISL goes down when portDiscovery property is disabled on the src/dst ports"() {
+        given: "A round trip latency ISL"
+        Isl roundTripIsl = topology.islsForActiveSwitches.find {
+            it.srcSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD) &&
+                    it.dstSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)
+        } ?: assumeTrue("Wasn't able to find a suitable link", false)
+
+        when: "Disable portDiscovery on the srcPort"
+        northboundV2.updatePortProperties(roundTripIsl.srcSwitch.dpId, roundTripIsl.srcPort,
+                new PortPropertiesDto(discoveryEnabled: false))
+        def portDiscoveryIsEnabledOnSrcPort = false
+
+        then: "Isl is still DISCOVERED"
+        Wrappers.wait(WAIT_OFFSET) {
+            def links = northbound.getAllLinks()
+            def islInfoForward = islUtils.getIslInfo(links, roundTripIsl,).get()
+            def islInfoReverse = islUtils.getIslInfo(links, roundTripIsl.reversed).get()
+            assert islInfoForward.state == DISCOVERED
+            assert islInfoForward.actualState == FAILED
+            assert islInfoReverse.state == DISCOVERED
+            assert islInfoReverse.actualState == DISCOVERED
+        }
+
+        when: "Disable portDiscovery property on the dstPort"
+        northboundV2.updatePortProperties(roundTripIsl.dstSwitch.dpId, roundTripIsl.dstPort,
+                new PortPropertiesDto(discoveryEnabled: false))
+        def portDiscoveryIsEnabledOnDstPort = false
+
+        then: "Status of the link is changed to FAILED"
+        //don't need to wait discoveryTimeout, disablePortDiscovery(on src/dst sides) == portDown
+        Wrappers.wait(WAIT_OFFSET) {
+            def allLinks = northbound.getAllLinks()
+            def islInfoForward = islUtils.getIslInfo(allLinks, roundTripIsl).get()
+            def islInfoReverse = islUtils.getIslInfo(allLinks, roundTripIsl.reversed).get()
+            assert islInfoForward.state == FAILED
+            assert islInfoForward.actualState == FAILED
+            assert islInfoReverse.state == FAILED
+            assert islInfoReverse.actualState == FAILED
+        }
+
+        when: "Enable portDiscovery on the src/dst ports"
```

Comment:
Maybe enable one by one? To see what happens if only 1-way traffic starts to flow